man/mlr_learners_surv.glmnet.Rd

```diff
@@ -24,7 +24,7 @@ lrn("surv.glmnet")
 }
 
 \examples{
-if (requireNamespace("glmnet")) {
+if (requireNamespace("glmnet survival")) {
```

Comment:
@RaphaelS1 This one here is unrelated to this PR but I just fell over it. Somehow multiple packages mentioned in `lrn$packages` are not picked up correctly by the roxygen template `example.R`. I took a quick look but got stuck.

I do not see {survival} being listed in the field, nevertheless it is still parsed in by roxygen somehow.

Reply:
mlr3proba dev fixed it, thanks!